scripts/lib.sh

```diff
@@ -1,5 +1,8 @@
 #!/bin/bash
 # Module containing code shared across various shell scripts
+# Execute functions from this module via the script do.sh
+
+alias pip="python -m pip --disable-pip-version-check"
```

Comment:
I think naming this as `pip` can be confusing? or maybe it doesn't matter since it's not doing much

Reply:
It's the same idea as what happens in the Makefile:
https://github.com/getsentry/sentry/blob/9c51aa4b768e7b273ec627983be8035f6e707477/Makefile#L1

It's to guarantee that every use of `pip` in the script will have the same behaviour and it does not persist outside of this call to the script.